lib/monitor.js

```diff
@@ -0,0 +1,43 @@
+'use babel'
+
+import log from './log'
+import commandLog from './command-log'
+
+function hostIP() {
+  return LEARN_IDE_HOST_IP
+}
+
+function clearHostIP() {
+  LEARN_IDE_HOST_IP = null
+}
+
+export default function monitor(terminal, subscriptions) {
+  subscriptions.add(atom.commands.onDidDispatch(({type}) => {
+    if (type === 'application:quit') { log.add({event: 'quit'}) }
```

Comment:
is it ever possible that the application quits before the post is made / can we guarantee this executes before the program exits?